installation: improve documentation and add easy to use install script (#200)
CodeIntelligenceTesting/cifuzz: .github/release_template.md

## Summary
**TODO**: If there are lot of changes in this release, please highlight the most important ones in this section

## How to install

> Remember to set the executable flag for the downloaded installer.
> You can do this by `chmod +x <installer file>`.

Reviewer: ```suggestion
On Linux and macOS, remember to make the downloaded installer executable via `chmod +x <installer file>`.
```
